### DISCUSSION TITLE
Plotting an scatterplot IPF of particle data

### DISCUSSION THREAD

**Discussion by tridevildog** [mtex-toolbox/mtex#562] [Ask Anything]
I am trying to create IPF's of some particles in an Aluminum Alloy.  I have the Euler angles of the particles saved in a .txt file, and I know the details of their crystallography.  I need help plotting IPFs of these particles.  I have found the online help and tutorials to be less than helpful. Can someone please help me get started? thanks.

**Comment by kilir** [comment]
Hi,

What "online help" are you referring to? 
Can you please elaborate what exactly in the documentation was less than helpful? Where did you got stuck?
And what is missing?

Cheers,
Rüdiger

PS: With respect to your question: What do you mean by "IPF of these particles"? You mean, you simply want to represent the orientation of these particles in an IPF?
Generic procedure: import your data and crystal symmetry and use orientation/plotIPDF().

**Comment by tridevildog** [comment]
Sir:

 

Thanks for your kind help.

 

A long time ago, in a galxy far, far away (about 10 years ago), I created these two pole figures:

 



And 

 



 

I created them in an old version of MTEX and MATLAB.

 

I need to replot them, while changing some of the closed circles to open circles, so that they can be published in a journal that does not allow color printing.

 

I have the Euler angles of all  of the points that I need plotted, and I have all of the crystallography data about the unit cells of the particles, but I have forgotten how I plotted it before, and the new MTEX looks nothing like the MTEX that I used before.

 

I have looked in the online documentation, but I can’t get anything to work like I think that it should. 

 

Any help would be greatly appreciated.

 

Respectfully Submitted

 

Robert McDaniels

Owner and Principal Investigator



Materials Research and Analysis Group of Tennessee

5987 Webb Lane

Murfreesboro, TN 37129

(800) 406-7248

(865) 680-2300

 <mailto:rmcdaniels@mragt.com> rmcdaniels@mragt.com

 <http://mragt.com/> http://mragt.com/

**Comment by kilir** [comment]
Here you should find the explanation on importing the orientations data:
https://mtex-toolbox.github.io/OrientationImport.html
If you don't use a CIF to define the crystal symmetry, have a look here:
https://mtex-toolbox.github.io/OrientationImport.html 
and here:
https://mtex-toolbox.github.io/crystalSymmetry.crystalSymmetry.html

For plotting IPFs see here:
https://mtex-toolbox.github.io/OrientationInversePoleFigure.html
and here:
https://mtex-toolbox.github.io/orientation.plotIPDF.html

Hope this helps.

Cheers,
Rüdiger
